### ISSUE TITLE
Windows Outlook only respects the first class on an element.

### ISSUE THREAD

**Issue by jkupczak** [hteumeuleu/email-bugs#75]
` 
<style>
  .a { color: red; }
  .b { color: blue; }
  .c { color: yellow; }
</style>

<div class="a b c">Hello world</div>
`

`Hello world` will be displayed in `red` instead of `yellow`.

**Comment by JayOram** [comment]
Hasn't this always been the case? I feel like it was a choice from the beginning, not a bug. Happy to be corrected and even happier if Outlook changed!

**Comment by hteumeuleu** [comment]
This has definitely always been the case. And for what it's worth, it was always how IE6 and below behaved (disrepecting standards specifications). I think it has its place here as it's definitely something quirky, specific to emails and Outlook, that non-email developers might struggle to find informations on.

**Comment by jkupczak** [comment]
> Hasn't this always been the case? I feel like it was a choice from the beginning, not a bug. Happy to be corrected and even happier if Outlook changed!

Yeah, this has always been the case. And as @hteumeuleu mentioned, it was like this in IE6 and below which the Word rendering engine is based on. I concede that this is not a bug in the traditional sense. But I think there are some things about Word/HTML4 rendering that should definitely be logged.

That said, there are several other quirks about coding for the Word engine. Not sure if this is a slippery slope in attempting to catalog them all as individual "bugs".

Two off the top of my head would be:

- Naturally inline elements can't be changed to block level.
- Block level elements can't be wrapped by an inline element (`<a>` around a `<table>` for example)

**Comment by SpecOps12** [comment]
ℹ️ I found out, that Windows 10 Office 365 Outlook is also just reading the first class.

**Comment by e1himself** [comment]
It's worth updating the title to also include Outlook 2021, along with Office 365 Outlook. As they do have this bug present as well.

**Comment by pbiolsi** [comment]
I assume from the example that this issue only effects styles that are not inlined… yes?



[https://www.kingarthurflour.com/sites/default/files/2020-07/king-arthur-baking-email-logo.png]<https://kingarthurbaking.com/bake-joy>  Peter Biolsi  he/him/his
Email Marketing Program Manager

+1 (802) 299-2240 Ext. 222

100% EMPLOYEE-OWNED



From: Ivan Voskoboinyk ***@***.***>
Date: Thursday, July 14, 2022 at 8:54 AM
To: hteumeuleu/email-bugs ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [hteumeuleu/email-bugs] Windows Outlook 07/10/13/16/19 only respects the first class on an element. (#75)
CAUTION: This email originated from outside of the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.


It's worth updating the title to also include Outlook 2021, along with Office 365 Outlook. As they do have this bug present as well.

**Comment by e1himself** [comment]
> I assume from the example that this issue only effects styles that are not inlined… yes?

Yes, that's correct. It only affects global stylesheets based on class selectors.
